packages/web-runtime/src/services/auth/userManager.ts

```diff
@@ -156,6 +158,10 @@ export class UserManager extends OidcUserManager {
       })
       this.initializeOwnCloudSdk(accessToken)
 
+      if (this.store.getters.capabilities?.core?.['support-sse']) {
+        ;(this.clientService.sseAuthenticated as SSEAdapter).updateAccessToken(accessToken)
```

Comment:
I still don't like this too much, but we tried to refactor it and it got complicated ... I'm ok as it seems to be the lesser evil for now

Reply:
Same here. I think we will need a tokenProvider and languageProvider in the client soon.